README.md

```diff
@@ -173,6 +173,30 @@ server "example.org" {
 
 Don't forget to `rcctl reload httpd` your configuration changes.
 
+### Docker
+
+Clone repo, alter gosh.yaml according to your needs and build container with contrib/docker/dockerfile
+
+or 
+
+use provided image from dockerhub 
```

Comment:
Thanks for providing a Docker image! However, the tagged version 0.6.0 is kinda outdated and lots have changed in the meantime, which I haven't completed into its own release, yet.

Maybe add a your username next to it, so that people are able to ping you for updates? Otherwise, this image should be built automatically through the CI.